anydex/test/restapi/test_wallets_endpoint.py

```diff
@@ -11,7 +11,7 @@ class TestWalletsEndpoint(TestRestApiBase):
     def setUp(self):
```

Comment:
We should probably adjust this method name to `set_up` to be PEP8 compliant. Makes me wonder how it passed the Tribler CI?

Reply:
I believe because those are "python defined" names. Like, if you name a method "setUp" in a test suite, python will know to execute it before every test and therefore I am not sure whether it can be changed.

Reply:
Ah I see, my bad. Missed the fact that the method was related to test setup.